README.md

```diff
@@ -41,6 +41,7 @@ A curated list of awesome open source workflow engines
 * [Piper](https://github.com/creactiviti/piper) [![Stars](https://img.shields.io/github/stars/creactiviti/piper.svg)](https://github.com/creactiviti/piper)  - A distributed Java workflow engine designed to be dead simple.
 * [Plynx](https://plynx.com) [![Stars](https://img.shields.io/github/stars/plynx-team/plynx.svg)](https://github.com/plynx-team/plynx) - Interactive platform with drag and drop interface for building and deploying portable and scalable end-to-end data driven workflows.
 * [Popper](https://github.com/systemslab/popper) [![Stars](https://img.shields.io/github/stars/systemslab/popper.svg)](https://github.com/systemslab/popper)  - Lightweight, YAML based container-native workflow engine supporting Docker, Singularity, Vagrant VMs with Docker daemon in VM, and local host.
+* [Parsl](https://parsl-project.org/) [![Stars](https://img.shields.io/github/stars/Parsl/parsl.svg)](https://github.com/Parsl/parsl) - Python framework for workflow orchestration and parallelization based on a dynamic graph of tasks and their data dependencies.
```

Comment:
plz put the row in the right alphabetical order in the list

Reply:
I must not have had my ☕ this morning!